Commands/SitesMassWordPressUpdateCommand.php

```diff
@@ -164,14 +178,41 @@ public function massWpUpdate($args, $assoc_args) {
       $this->send_to_slack("Updating {$sites_count} websites ...");
     }
 
+    $excludedSites = array();
+
+    if ($this->yaml_settings) {
+
+      $excludedSites = $this->yamlGetExcludedSites($this->yaml_settings);
+
+    }
+
     // Loop through each site and update.
-    foreach ($sites as $site) {
-      $args = array(
-        'name'      => $site->get('name'),
-        'env'       => $env,
-        'framework' => $site->attributes->framework,
-      );
-      $this->update($args, $assoc_args);
+    foreach ( $sites as $site ) {
+
+      if ( ! in_array( $site->get('name'), $excludedSites ) ) {
+
+        $args = array(
+          'name'      => $site->get('name'),
+          'env'       => $env,
+          'framework' => $site->attributes->framework,
+        );
+
+        if ( $this->yaml_settings ) {
+
+          $site_args = $this->yamlGetSiteArgs( $site->get('name'), $this->yaml_settings );
+
+          $args = ( $site_args ? $site_args : $args );
+
```

Comment:
You should merge`$site_args` with `$args` so we don't have to place the `framework` in the yml file, use `$args` as a default